_projects/2022/sunpy/database.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Database as a data client
+desc: Creating a new database module
+requirements:
+ - Familiarity with Python
+difficulty: high
+issues:
+- https://github.com/sunpy/sunpy/issues?q=is%3Aissue+is%3Aopen+database
+mentors:
+ - cadair
+ - nabobalis
+initiatives:
+ - GSOC
+project_size:
+ - medium
+ - large
```

Comment:
I was unable to decide.

Reply:
if it would work with either scope I think it's OK to keep both. We were told that if we go for large project only we can expect fewer slots than if it's a mix, etc. (We very much don't know yet how this all will work with all the changes to the student selection this year).